librosa/effects.py

```diff
@@ -568,3 +570,74 @@ def split(y, top_db=60, ref=np.max, frame_length=2048, hop_length=512):
 
     # Stack the results back as an ndarray
     return edges.reshape((-1, 2))
+
+
+def preemphasis(y, coef=0.97, zi=None, return_zf=False):
+    '''Pre-emphasize (high-pass filter) an audio signal:
```

Comment:
'''Pre-emphasize an audio signal with a first-order autoregressive filter:

(note that there is nothing preventing `coef` to be negative, in which case the filter becomes low-pass. bug or feature?)